scripts/packages/VSCodeServer/src/eval.jl

```diff
@@ -279,16 +279,19 @@ end
 const LOCATION_REGEX = r"\[\d+\]\s(?<body>.+)\sat\s(?<path>.+)\:(?<line>\d+)"
 
 function backtrace_string(bt)
-    s = sprintlimited(bt, func = Base.show_backtrace, limit = MAX_RESULT_LENGTH)
-    lines = strip.(split(s, '\n'))
-
-    return join(map(enumerate(lines)) do (i, line)
-        i === 1 && return line # "Stacktrace:"
-        m = match(LOCATION_REGEX, line)
```

Comment:
I think we can remove [the definition of `LOCATION_REGEX`](https://github.com/julia-vscode/julia-vscode/blob/a84b6def506dfad9ce81e2eefaf637597c384a23/scripts/packages/VSCodeServer/src/eval.jl#L278-L279).

Reply:
Done.